src/content/docs/new-relic-solutions/business-observability/configure-pathpoint.mdx

```diff
@@ -65,104 +63,96 @@ There are three icons that will let you access each of the three main editors in
   alt="TouchPoint editor"
   src={touchpointEditor}
 />
-## What is a JSON configuration file? [#json-config]
 
-JSON (JavaScript Object Notation) is a data structure, whose basic function is to allow for the exchange of information. Through this structure, you can identify each of the elements and components that will facilitate the implementation of Pathpoint. This includes understanding the function of its attributes, queries, and data output. 
+## Using the JSON interchange format
 
-<img
-  title="JSON file"
-  alt="JSON file"
-  src={jsonfile}
-/>
+It is extremely easy and intuitive to construct a Pathpoint implementatino using the stages, steps, and touchpoints editors shown in the previous section.  Sometimes it will be useful or necessary to store the configuration in a file and import that configuration at a later time possibly in another account or Pathpoint instance.  For a production Pathpoint implementation we strongly recommend storing an archive of your configuration as JSON in a version control system such as git.
```

Comment:
```suggestion
It is extremely easy and intuitive to construct a Pathpoint implementation using the stages, steps, and touchpoints editors shown in the previous section.  Sometimes it will be useful or necessary to store the configuration in a file and import that configuration at a later time possibly in another account or Pathpoint instance.  For a production Pathpoint implementation we strongly recommend storing an archive of your configuration as JSON in a version control system such as git.
```